src/layouts/Login/LoginPage.tsx

```diff
@@ -87,7 +90,7 @@ const LoginContent = (): JSX.Element => {
 
   const handleVerifyOtp = async ({ otp }: OtpProps) => {
     await verifyLoginOtp({ email, otp })
-    window.location.reload()
+    history.replace("/sites")
```

Comment:
i think we might've missed removing this!

Reply:
It's still necessary here - the onSuccess only checks and sets the cookie, unless we want to move the history modification into the hook as well?

Reply:
oh my b - you're right. i've no strong opinion here